commit contacto

MarianoSaez opened the pull request.

IvanDerlich commented:
probando

IvanDerlich commented:
probando2

IvanDerlich commented:
probando3

jfenderico commented:
Pensé que habías creado el directorio, pero veo en el log que luego lo borraste. Para que haga un merge del pull request, deberías volver a crear el directorio con tu nro de legajo.

IvanDerlich commented:
ahi lo cree, pero no se como agregarlo al proyecto pricipal ( que veo que no esta)

jfenderico commented:
Debería contener un archivo, aunque sea vacío, para que puedas subir el dir a tu cuenta de github y luego crear el pull request.

jfenderico commented:
La primera vez lo hiciste bien, pero luego borraste el directorio al borrar el txt que habías puesto dentro. Repetí los mismo pasos.

IvanDerlich commented:
en este punto estoy re perdido, no cazo un fobal

IvanDerlich commented:
entiendo el problema, no se como solucionarlo

jfenderico commented:
Lo vemos mañana.

Commit:
commit contacto

MarianoSaez committed:
Merge pull request #1 from umcomp2/main

update (creo)

MarianoSaez committed:
Merge pull request #2 from umcomp2/main

update_clase-4

MarianoSaez committed:
Merge pull request #3 from umcomp2/main

update clase 5

MarianoSaez committed:
Merge pull request #4 from umcomp2/main

update

MarianoSaez committed:
1st commit

MarianoSaez committed:
manejo de errores

MarianoSaez committed:
testcase sin extension

MarianoSaez committed:
mas y mas error handling

MarianoSaez committed:
moved

MarianoSaez commented:
tp1

MarianoSaez committed:
Merge pull request #5 from umcomp2/main

update

MarianoSaez committed:
Merge pull request #6 from umcomp2/main

update

MarianoSaez committed:
Merge pull request #7 from umcomp2/main

Enunciado TP2 - entrega 15/05/2021

MarianoSaez committed:
tp2-1st commit

MarianoSaez committed:
matrix filling

MarianoSaez committed:
Merge pull request #8 from umcomp2/main

mas imagenes ppm

MarianoSaez committed:
optimizacion de escritura

MarianoSaez committed:
tp2 a prueba de casi todo creo xd

MarianoSaez committed:
ultimo commit tp2

MarianoSaez commented:
tp2

MarianoSaez committed:
intro sockets cliente-servidor

MarianoSaez committed:
close fix

MarianoSaez committed:
wrapper de protocolo

MarianoSaez committed:
cliente_juncotic

MarianoSaez committed:
remote_shell_multicli

MarianoSaez committed:
remote_shell_pickle

MarianoSaez committed:
comienzo tcp_udp

MarianoSaez committed:
tcp_udp

MarianoSaez committed:
Implementacion del remote_shell_pickle usando el modulo socketserver

Consiste en reimplementar el codigo de remote_shell_pickle con el
framework provisto en la libreria estandar de python socketserver

MarianoSaez committed:
chained

MarianoSaez committed:
remote_shell_asyncio

MarianoSaez committed:
celery_remote_calc
